src/Adapter/PublicCDNAdapter.php

```diff
@@ -15,9 +16,12 @@ class PublicCDNAdapter extends PublicAdapter implements SilverstripePublicAdapte
 {
     protected $cdnPrefix;
 
-    public function __construct(S3Client $client, $bucket, $prefix = '', $cdnPrefix = '', array $options = [])
+    protected $cdnAssetPath;
+
+    public function __construct(S3Client $client, $bucket, $prefix = '', $cdnPrefix = '', $cdnAssetPath = '', array $options = [])
```

Comment:
I think adding a new constructor param in the middle of the list will require a new major release.

Reply:
Have added it to last place to avoid the issue with versioning and updated the README.md as well with its correct position after options. Makes sense, I did consider it, but was only thinking within the context of the repo's use of the class.